README.md

```diff
@@ -22,17 +22,17 @@ Example using a pinned version:
 
 ```yaml
 jobs:
-  test_cosign_action:
+  example:
     runs-on: ubuntu-latest
 
     permissions: {}
 
-    name: Install Cosign and test presence in path
+    name: Install Cosign
     steps:
       - name: Install Cosign
-        uses: sigstore/cosign-installer@main
+        uses: sigstore/cosign-installer@v3.0.5
         with:
-          cosign-release: 'v2.0.2'
+          cosign-release: 'v2.0.0'
```

Comment:
We should recommend 2.0.2 as it contains the TUF root fetched over cdn

Reply:
ok, that was just an example, but will update

Reply:
done, PTAL

Reply:
Understood, but since people will copy and paste this id rather not recommend an old version